### ISSUE TITLE
docker build errors

### ISSUE THREAD

**Issue by iandow** [trask/heatclinic-demo#2]
I'm getting the following error when I try to build this docker image:

```
Caused by: java.lang.ClassNotFoundException: ${database.driver}
```

Also, it looks like the glowroot-dist.zip download fails with this command:

```
curl https://glowroot.s3.amazonaws.com/snapshots/latest/glowroot-dist.zip
```

**Comment by trask** [comment]
Thanks.  I think I fixed in 08cba57c1443aca12bd730166bc431dd626812ed.  Can you try again?

**Comment by iandow** [comment]
Yup. It works now. I can build and run the image.  Thanks for the fix.
